app/models/manageiq/providers/amazon/regions.rb

```diff
@@ -1,92 +1,15 @@
-          :hostname    => "ec2.us-east-2.amazonaws.com",
-          :description => "US East (Ohio)",
-        },
-        "us-west-1"      => {
-          :name        => "us-west-1",
-          :hostname    => "ec2.us-west-1.amazonaws.com",
-          :description => "US West (Northern California)",
-        },
-        "us-west-2"      => {
-          :name        => "us-west-2",
-          :hostname    => "ec2.us-west-2.amazonaws.com",
-          :description => "US West (Oregon)",
-        },
-        "eu-west-1"      => {
-          :name        => "eu-west-1",
-          :hostname    => "ec2.eu-west-1.amazonaws.com",
-          :description => "EU (Ireland)",
-        },
-        "eu-west-2"      => {
-          :name        => "eu-west-2",
-          :hostname    => "ec2.eu-west-2.amazonaws.com",
-          :description => "EU (London)",
-        },
-        "eu-west-3"      => {
-          :name        => "eu-west-3",
-          :hostname    => "ec2.eu-west-3.amazonaws.com",
-          :description => "EU (Paris)",
-        },
-        "eu-central-1"   => {
-          :name        => "eu-central-1",
-          :hostname    => "ec2.eu-central-1.amazonaws.com",
-          :description => "EU (Frankfurt)",
-        },
-        "ap-south-1"     => {
-          :name        => "ap-south-1",
-          :hostname    => "ec2.ap-south-1.amazonaws.com",
-          :description => "Asia Pacific (Mumbai)",
-        },
-        "ap-southeast-1" => {
-          :name        => "ap-southeast-1",
-          :hostname    => "ec2.ap-southeast-1.amazonaws.com",
-          :description => "Asia Pacific (Singapore)",
-        },
-        "ap-southeast-2" => {
-          :name        => "ap-southeast-2",
-          :hostname    => "ec2.ap-southeast-2.amazonaws.com",
-          :description => "Asia Pacific (Sydney)",
-        },
-        "ap-northeast-1" => {
-          :name        => "ap-northeast-1",
-          :hostname    => "ec2.ap-northeast-1.amazonaws.com",
-          :description => "Asia Pacific (Tokyo)",
-        },
-        "ca-central-1"   => {
-          :name        => "ca-central-1",
-          :hostname    => "ec2.ca-central-1.amazonaws.com",
-          :description => "Canada (Central)",
-        },
-        "ap-northeast-2" => {
-          :name        => "ap-northeast-2",
-          :hostname    => "ec2.ap-northeast-2.amazonaws.com",
-          :description => "Asia Pacific (Seoul)",
-        },
-        "sa-east-1"      => {
-          :name        => "sa-east-1",
-          :hostname    => "ec2.sa-east-1.amazonaws.com",
-          :description => "South America (Sao Paulo)",
-        },
-        "us-gov-west-1"  => {
-          :name        => "us-gov-west-1",
-          :hostname    => "ec2.us-gov-west-1.amazonaws.com",
-          :description => "GovCloud (US)",
-        }
-      }
+      REGIONS = YAML.load_file(
+        ManageIQ::Providers::Amazon::Engine.root / 'lib' / 'data' / 'aws_regions.yaml'
```

Comment:
Prefer the normal method approach as opposed to the `/` method for consistency...

```ruby
ManageIQ::Providers::Amazon::Engine.root.join("lib/data/aws_regions.yaml")
```

Reply:
Why `lib/data`?  The lib directory is normally used for source.  Perhaps config directory is better as that's where we typically store yml files?  Also, prefer yml over yaml as the extension for consistency.

Reply:
Can you please point me to some similar file? I believe that **data** have nothing common with **configuration**. If you don't like `lib` directory, let's discuss what place will be good for a data files. It can be somewhat like `db/fixtures`, but current gem doesn't have it, should I create?

Reply:
It's an interesting distinction. I guess `db/fixtures` works as a directory.  That would be consistent with the main repo at least.